plugins/inlinetoolbar/plugin.js

```diff
@@ -32,13 +32,34 @@
 		this._listeners = [];
 	};
 
+	/**
+	 * Class representing instance of inline toolbar.
+	 *
+	 * @class
+	 * @constructor Creates an inline toolbar instance.
+	 * @since 4.8
+	 * @param {CKEDITOR.editor} editor The editor instance for which the panel is created.
+	 * @param {Object} definition An object containing the panel definition. See {@link CKEDITOR.ui.balloonPanel}
+	 * docs for an example definition.
+	 */
+	CKEDITOR.ui.inlineToolbar = function( editor, definition ) {
+		this._toolbar = new CKEDITOR.ui.inlineToolbarView( editor, definition );
+		/**
+		 * Menu items added to inline toolbar
+		 *
+		 * @private
+		 */
+		this._menuItems = [];
```

Comment:
I think we might change this name simply to `_items` as it's not a menu, it's a toolbar.